package-lock.json

```diff
@@ -0,0 +1,10266 @@
+{
```

Comment:
Looks like this slipped into your last commit. (This project already uses yarn for a lockfile.)

Reply:
You're right, sorry about that.